src/main/webapp/i18n/de/exam.json

```diff
@@ -301,6 +301,7 @@
             "confirmationEndText": "Bestätigungstext zum Klausurende",
             "assessmentDashboard": "Bewertung",
             "examChecklist": "Klausurcheckliste",
+            "examMonitoring": "Klausurüberwachung",
```

Comment:
this translation does not sound good to me. We should change this to exam live statistics in English an Klausur Live Statistik in German.

Reply:
Good idea - sounds better! Thank you :+1: